### DISCUSSION TITLE
Getting TypeError while using Grapesjs with React

### DISCUSSION THREAD

**Discussion by theashishmaurya** [GrapesJS/grapesjs#4525] [Q&A]
**How to reproduce the bug?**
1. Install Grapesjs with any react version 18.2.0 and it will throw the error in the console.
here is a link to reproduce it. 
[Sandbox](https://codesandbox.io/s/falling-snow-dion3q?file=/src/App.tsx)

**What is the expected behavior?**
It should be loading the div inside the component.

...

**What is the current behavior?**
Throwing TypeError: m.render is not a function

![image](https://user-images.githubusercontent.com/50199029/186091810-455bbbc8-b2dc-48f9-a4a4-3bbe82252ff3.png)

...
The code I used to initialize the Grapesjs
```tsx
import "./styles.css";
import 'grapesjs/dist/css/grapes.min.css';
import { useEffect } from "react";
import grapesjs from "grapesjs"


export default function App() {

  useEffect(() => {
    console.log("Init");
    const editor = grapesjs.init({
      container: "#gjs",
      fromElement: true,
      height: "300px",
      width: "auto",
      storageManager: false,
      panels: { defaults: [] },
    });

  }, []);

  return (
    <div id="gjs">
      <h1>Hello CodeSandbox</h1>
      <h2>Start editing to see some magic happen!</h2>
    </div>
  );
}


```

**Comment by jmaster92** [comment]
Hello, i have the same problem and i tried many ways but nothing,  did you found a solution to this? regards

**Comment by artf** [comment]
I guess that might be similar to this one https://github.com/artf/grapesjs/discussions/4565#discussioncomment-3656352
